docs/src/main/asciidoc/doc-contribute-docs-howto.adoc

```diff
@@ -36,10 +36,11 @@ To ensure that your content shows up correctly on the link:https://quarkus.io/gu
 . Decide on a content type that best fits the content that you are contributing.
```

Comment:
While you are editing this topic... I think it would be good to mention diataxis and link to the detailed concept topic on the Quarkus content types here. WDYT? Thanks :-) 

```suggestion
. Decide on a [diataxis](https://quarkus.io/guides/doc-concept) content type that best fits the content that you are contributing.
```

Reply:
Updated. Thanks.